expression/builtin_string_test.go

```diff
@@ -1101,19 +1101,12 @@ func (s *testEvaluatorSuite) TestChar(c *C) {
 		}
 	}
 
-	v := struct {
-		str    string
-		iNum   int64
-		fNum   interface{}
-		result string
-	}{"65", 66, nil, "AB"}
```

Comment:
Why remove this?

Reply:
I don't remove it, just modify it, to make go lint happy.